api/src/services/graphql/index.ts

```diff
@@ -851,6 +851,8 @@ export class GraphQLService {
 				ReadableCollectionFilterTypes[collection.collection].addFields({
 					_and: [ReadableCollectionFilterTypes[collection.collection]],
 					_or: [ReadableCollectionFilterTypes[collection.collection]],
+					_none: [ReadableCollectionFilterTypes[collection.collection]],
+					_some: [ReadableCollectionFilterTypes[collection.collection]],
```

Comment:
`none` and `some` are only available as the first level nesting of a O2M field, like:

```
{
  _and: [
    {
      title: {
        _eq: 'Example'
      }
    },
    {
      favorites: {
        _none: { <<<<<<<<<<<<<<<
          rating: {
            _lte: 3
          }
        }
      }
    }
  ]
}

Reply:
Ok, will come back to this one and make sure we're only using it in the right place after the I push the other PR 👍🏻